src/components/Party/Party.vue

```diff
@@ -35,6 +36,7 @@ export default {
   },
   data() {
     return {
+      partyId : String(this.$route.params.partyId),
```

Comment:
Is "String()" really necessary ?

Reply:
Inspired by CreateParty and JoinPartyStep1, apparently not necessary for this case